### PR TITLE
Add Angular2 Directive example

### PR DESCRIPTION
**Pull request by leocaseiro**
Usage:

```html
<button toggleScreenfull></button>
```

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -158,6 +158,28 @@ You can use the [Angular.js binding](https://github.com/hrajchert/angular-screen
 </div>
 ```
 
+#### Toggle Fullscren as an Angular 2 Directive
+
+```typescript
+import { Directive, HostListener } from '@angular/core';
+const screenfull = require('screenfull/dist/screenfull');
```
Comment (sindresorhus) [inline readme.md]: Should just be:

```js
const screenfull = require('screenfull');
```

The `main` entry is defined to be the dist.

File: readme.md [hunk 2]
```diff
@@ -158,6 +158,28 @@ You can use the [Angular.js binding](https://github.com/hrajchert/angular-screen
...
+
+@Directive({
+    selector: '[toggleScreenfull]'
+})
+export class ToggleScreenfullDirective {
+    @HostListener('click') onClick() {
```
Comment (sindresorhus) [inline readme.md]: `onClick()` should probably be on a newline to make it clearer the above is a decorator.

File: readme.md [hunk 3]
```diff
@@ -158,6 +158,28 @@ You can use the [Angular.js binding](https://github.com/hrajchert/angular-screen
...
+        if (screenfull.enabled) {
+            screenfull.toggle();
+        }
+    }
+}
+```
+
+```html
+<button toggleScreenfull>Toggle Fullscreen<button>
```
Comment (sindresorhus) [inline readme.md]: I would go with `toggleScreenfull`=>`toggleFullscreen` here as it's more readable.

File: readme.md (hunk 2 above)
Comment (leocaseiro) [inline readme.md]: I believe that's how the Angular Style Guide use attribute decorators: https://angular.io/styleguide#!#06-01

File: readme.md (hunk 3 above)
Comment (leocaseiro) [inline readme.md]: I was trying to follow the npm lib name. But'll change it

File: readme.md [hunk 4]
```diff
@@ -158,6 +158,28 @@ You can use the [Angular.js binding](https://github.com/hrajchert/angular-screen
 </div>
 ```
 
+#### Toggle Fullscren as an Angular 2 Directive
```
Comment (sindresorhus) [inline readme.md]: The title should be consistent with the Angular 1 example title. Either change the other one or this.

Comment (leocaseiro) [inline readme.md]: I changed to Fullscreen the page with Angular 2 as my example is valid for a full page, not an element

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by oztek22** [comment]
not working with angular2

**Comment by leocaseiro** [comment]
@oztek22 what's the error are you getting?

If you are using `angular-cli`, try:

```typescript
import { Directive, HostListener } from '@angular/core';
import * as screenfull from 'screenfull';

@Directive({
    selector: '[toggleScreenfull]'
})
export class ToggleScreenfull {
    @HostListener('click') onClick() {
        if (screenfull.enabled) {
            screenfull.toggle();
        }
    }
}
```


Usage:

```
<button toggleScreenfull></button>
```

**Comment by oztek22** [comment]
@leocaseiro thx for help. actually I declared ToggleFullscreenDirective in wrong module that's why I was getting nothing. ':)

**Pull request merged**